tests/check_array.c

```diff
@@ -27,20 +27,20 @@ START_TEST(test_take1_ok) {
                               18, 19, 20, 21, 22, 23};
   uint32_t buf[12 * 24];
   const size_t data_dims[1] = {24};
-  ssize_t indexes[12];
```

Comment:
That is groundwork for call32.  Also I didn't remove all uses of the type, just the ability to make arrays, because arrays of a type that can change size is a bad idea.